docs/devel/CONTRIBUTING.md

```diff
@@ -317,12 +312,13 @@ Please refer to the Kinvolk
 For making PRs/commits consistent and easier to review, please check out
 Kinvolk's [contribution guidelines on git](https://github.com/kinvolk/contribution/blob/master/topics/git.md).
 
+We require all commits on a PR to be signed off.
```

Comment:
not sure if you want to point to the DCO definition https://developercertificate.org/

Reply:
Done!